.github/workflows/build-and-push.yml

```diff
@@ -35,16 +57,40 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
 
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v2
+      uses: docker/setup-buildx-action@v3
```

Comment:
I don't think we need buildx if we're building on native architecture! buildx just sets up the emulation tools!